crates/sui-types/src/gas.rs

```diff
@@ -239,9 +239,6 @@ pub struct SuiCostTable {
     /// Unit cost of a byte in the storage. This will be used both for charging for
     /// new storage as well as rebating for deleting storage. That is, we expect users to
     /// get full refund on the object storage when it's deleted.
-    /// TODO: We should introduce a flat fee on storage that does not get refunded even
-    /// when objects are deleted. This cost covers the cost of storing transaction metadata
```

Comment:
Yes this comment should not be there anymore; it’s leftover from when we tried flat fees.

Reply:
Sorry I commented on the wrong change.